src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  let sum = 0;
+
+  if (days >= 1) {
+    sum = days * value;
+  }
+
+  if (days >= 3) {
+    sum = days * value - 20;
+  }
+
+  if (days >= 7) {
+    sum = days * value - 50;
```

Comment:
move all magic numbers(20, 50) to variables

also, you can return values immediately inside if statements